pkg/lib/authenticationflow/declarative/intent_signup_flow.go

```diff
@@ -126,6 +126,15 @@ func (i *IntentSignupFlow) GetEffects(ctx context.Context, deps *authflow.Depend
 			}
 			return nil
 		}),
+		authflow.OnCommitEffect(func(ctx context.Context, deps *authflow.Dependencies) error {
+			userID, createUser := i.userID(flows)
+			if !createUser {
+				return nil
+			}
```

Comment:
Even if the user is not created (I think it occurs in account linking), the login time should still be updated

Reply:
Actually I followed the logic of the below effect. When it is account linking, the below effect (the one that calls deps.Users.AfterCreate, which should in turn dispatch user.created) is skipped.

If we update login time, then we will introduce the same problem again: Login time was updated but no audit log events are dispatched.

Reply:
I see, in account linking we have a login flow inside a signup flow so that flow will update the login time